tests/python/frontend/pytorch/test_forward.py

```diff
@@ -5355,6 +5355,16 @@ def forward(self, *args):
         assert "%aten::_convolution_0" in graph
 
 
+def test_inplace_copy():
+    class InplaceCopy(torch.nn.Module):
+        def forward(self, x):
+            x[:5, 0] = x[:5, 0] + 1
+            return x
+
+    inputs = torch.randn(10, 10)
+    verify_model(InplaceCopy(), [inputs])
```

Comment:
Please add more tests, using various tricky examples to make sure that the conversion works.

Reply:
I added more test, tried to test this function with various case. Please let me know about any suggestions

Reply:
thanks